internal/pkg/service/common/etcdop/op/txn.go

```diff
@@ -140,7 +140,7 @@ func (v *TxnOpDef) Txn(ctx context.Context) *TxnOp {
 		txn.Else(newInlineOp(etcd.OpTxn(subCmps, []etcd.Op{}, subElse), nil, item.MapResponse))
 	}
 
-	txn.processors = v.processors[:]
+	txn.processors = v.processors
```

Comment:
There was (and is) logical error, slice should be copied, so `v.processors` and `txn.processors` are decoupled.

We can discuss it on standup call.

```
txn.processors = append([]txnProcessor(nil), v.processors...) // (copy slice)
```

Reply:
(I didn't realize that `[:]` opeartor doesn't copy slices either.)

Reply:
Yes, therefore, it is always safe to append only to the same variable, either a new slice is created or the existing unused capacity in the old slice is used.

This validates the linter you turned on. It can be simply forgotten.

This characteristic of slices is intensively used, for example, in buffers. Memory allocation is an expensive operation.

Reply:
```
txn.processors = append([]txnProcessor(nil), v.processors...)
```